webAO/client.css

```diff
@@ -421,6 +424,10 @@ img {
 	filter: brightness(50%);
 }
 
+.noini {
+	transform: rotate(180deg);
```

Comment:
this might be a little too subtle for most people to understand...

Reply:
suggestions on how to do it instead?
darkening them makes them look like in use...

Reply:
this is More Work(tm), but maybe add a warning icon on the bottom-left or bottom-right of the picture, and when the user hovers it, it says that "the character may not be usable due to missing files"